quanda/metrics/heuristics/mixed_datasets.py

```diff
@@ -1,6 +1,6 @@
 """Mixed Datasets Metric."""
 
-from typing import Any, List, Optional, Union, Callable
+from typing import Any, Callable, List, Optional, Union
```

Comment:
Which formatter do you use? The one we have in the Makefile under `clean-format` doesn't do import sorting, but it would be nice if it did. Is it a `ruff` setting?